types/dummy/index.d.ts

```diff
@@ -0,0 +1,3 @@
+// FIXME: Not sure this actually does anything
```

Comment:
See https://github.com/emberjs/ember-test-helpers/pull/1272/files#diff-a13bad01b7436c90fb6e3eafacf9947547a2fb96f8abc38588b1f06fcc7e2d55

Reply:
This definitely *should* do a thing, but it depends on how `paths` is set I think. This may be basically-invisible as it stands; this is a problem with our blueprint and config that has existed for a very long time.